content/doc/tutorials/_new_prerequisites.adoc

```diff
@@ -0,0 +1,14 @@
+////
+This file is only meant to be included as a snippet in other documents.
+////
+
+=== Prerequisites
+
+For this tutorial, you will require:
+
+* A macOS, Linux or Windows machine with:
+** 2 GB of RAM
+** A few GB of drive space for Jenkins
```

Comment:
Having a concrete number here would be good to have so there are no misinterpretations. If it will not need all 10gbs that had been in the previous prerequisites, is it cutting that size in half, more, or less?

Reply:
You're right. 
I think 2GB would be more than enough (in fact, I would be tempted to write a few hundred megabytes, but...).